os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -1063,28 +1064,29 @@ bool libspdm_gen_csr(uint32_t base_hash_algo, uint32_t base_asym_algo, bool *nee
     size_t asym_nid;
     void *context;
 
-    uint8_t *cached_req_info;
-    size_t cached_req_info_length;
+    uint8_t *cached_last_csr_request;
+    size_t cached_last_request_len;
     uint8_t *cached_csr;
     size_t csr_buffer_size;
 
     csr_buffer_size = *csr_len;
 
     /*device gen csr need reset*/
     if (*need_reset) {
```

Comment:
`*need_reset` is an output from `libspdm_gen_csr`. Why is it evaluating it as an input?

Reply:
agree. It should be output only.

Reply:
OK. Then we can merge this one.

Reply:
Yes. There is a unit_test to cover the check.